diesel/src/pg/backend.rs

```diff
@@ -48,7 +47,7 @@ impl<'a> PgValue<'a> {
 
     /// Get the type oid of the value represented by this raw value
     pub fn oid(&self) -> u32 {
```

Comment:
I think just returning `Option<NonZeroU32>` here is the better solution, so we can handle that at the calling side, because it may be ok to have no oid.

Reply:
```suggestion
    pub fn oid(&self) -> Option<NonZeroU32> {
```